<!DOCTYPE html>
<html lang="en-us">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="theme" content="hugo-academic">
  <meta name="generator" content="Hugo 0.40.1" />
  <meta name="author" content="Yixian Huang(黄奕衔)">
  

  
  
  
    
  
  
    
    
    <link rel="stylesheet" href="/css/highlight.min.css">
    
  
  
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha512-6MXa8B6uaO18Hid6blRMetEIoPqHf7Ux1tnyIQdpt9qI5OACx7C+O3IVTr98vwGnlcg0LOLa02i9Y1HpVhlfiw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.0/css/academicons.min.css" integrity="sha512-GGGNUPDhnG8LEAEDsjqYIQns+Gu8RBs4j5XGlxl7UfRaZBhCCm5jenJkeJL8uPuOXGqgl8/H1gjlWQDRjd3cUQ==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha512-SfTiTlX6kk+qitfevl/7LibUOeJWlt9rbyDn92a1DqWOw9vWG2MFoays0sgObmWazO5BQPiFucnnEAjpAB+/Sw==" crossorigin="anonymous">
  
  
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:400,700%7CMerriweather%7CRoboto+Mono">
  <link rel="stylesheet" href="/css/hugo-academic.css">
  

  
    <script>
      window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
      ga('create', 'UA-102569576-1', 'auto');
      ga('require', 'eventTracker');
      ga('require', 'outboundLinkTracker');
      ga('require', 'urlChangeTracker');
      ga('send', 'pageview');
    </script>
    <script async src="//www.google-analytics.com/analytics.js"></script>
    
    <script async src="https://cdnjs.cloudflare.com/ajax/libs/autotrack/2.4.1/autotrack.js" integrity="sha512-HUmooslVKj4m6OBu0OgzjXXr+QuFYy/k7eLI5jdeEy/F4RSgMn6XRWRGkFi5IFaFgy7uFTkegp3Z0XnJf3Jq+g==" crossorigin="anonymous"></script>
    
  

  <link rel="alternate" href="https://huangyixian666.github.io/index.xml" type="application/rss+xml" title="Yixian HUANG">
  <link rel="feed" href="https://huangyixian666.github.io/index.xml" type="application/rss+xml" title="Yixian HUANG">

  <link rel="icon" type="image/png" href="/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/apple-touch-icon.png">

  <link rel="canonical" href="https://huangyixian666.github.io/">

  

  <title>Yixian Huang</title>

</head>
<body id="top" data-spy="scroll" data-target="#navbar-main" data-offset="71">

<nav class="navbar navbar-default navbar-fixed-top" id="navbar-main">
  <div class="container">

    
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
              data-target=".navbar-collapse" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="/">Yixian Huang</a>
    </div>

    
    <div class="collapse navbar-collapse">

      
      <ul class="nav navbar-nav navbar-right">
        

        

        <li class="nav-item">
          <a href="/#about" data-target="#about">
            
            <span>Home</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#publications_selected" data-target="#publications_selected">
            
            <span>Publications</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#posts" data-target="#posts">
            
            <span>Awards</span>
          </a>
        </li>

        
        

        

        <li class="nav-item">
          <a href="/#contact" data-target="#contact">
            
            <span>Contact</span>
          </a>
        </li>

        
        

        
      </ul>

    </div>
  </div>
</nav>



<span id="homepage" style="display: none"></span>



  
  
  <section id="about" class="home-section">
    <div class="container">
      



<div class="row" itemprop="author" itemscope itemtype="http://schema.org/Person" itemref="person-email person-address">
  <div class="col-xs-12 col-md-4">
    <div id="profile">

      
      <div class="portrait" style="background-image: url('https://huangyixian666.github.io/img/portrait.jpg');"></div>
      <meta itemprop="image" content="https://huangyixian666.github.io/img/portrait.jpg">
      

      <div class="portrait-title">
        <h2 itemprop="name">Yixian Huang(黄奕衔)</h2>
        <h3 itemprop="jobTitle"></h3>
        
        <h3 itemprop="worksFor" itemscope itemtype="http://schema.org/Organization">
          
            <span itemprop="name">School of Medicine, The Chinese University of Hong Kong, Shenzhen (CUHKSZ)</span>
          
        </h3>
        
      </div>

      <link itemprop="url" href="https://huangyixian666.github.io/">

      <ul class="social-icon" aria-hidden="true">
        
        
        <li>
          <a itemprop="sameAs" href="http://orcid.org/0000-0001-8283-7274" target="_blank">
            <i class="ai ai-orcid big-icon"></i>
          </a>
        </li>
        
        
        <li>
          <a itemprop="sameAs" href="https://scholar.google.com/citations?user=wtQnq4AAAAAJ&amp;hl=en" target="_blank">
            <i class="ai ai-google-scholar big-icon"></i>
          </a>
        </li>
        
        
        <li>
          <a itemprop="sameAs" href="https://www.researchgate.net/profile/Taosheng_Xu" target="_blank">
            <i class="ai ai-researchgate big-icon"></i>
          </a>
        </li>
        
        
        <li>
          <a itemprop="sameAs" href="https://github.com/taoshengxu" target="_blank">
            <i class="fa fa-github big-icon"></i>
          </a>
        </li>
        
      </ul>

    </div>
  </div>
  <div class="col-xs-12 col-md-8" itemprop="description">

    <p>Yixian Huang is currently a PhD student at the Arieh Warshel Institute for Computational Biology (Led by Nobel laureate in Chemistry in 2013 Prof. Arieh Warshel), School of Medicine, The Chinese University of Hong Kong, Shenzhen (CUHKSZ). He received his Bachelor Degree in School of Life Science from Sun Yat-sen University (SYSU). His research interests include Bioinformatics, Biological Big Data Platforms and Artificial Intelligence Drug Development (AIDD). Mainly focus on the development of innovative methods for drug target identification and drug repurposing platforms based on biological multi-omics data and artificial intelligence technology, and use the developed methods to explore the treatment of Sorafenib-resistant Hepatocellular Carcinoma and conduct experimental verification .</p>


    <div class="row">

      
      <div class="col-sm-5">
        <h3>Research Interests</h3>
        <ul class="ul-interests">
          
          <li>Bioinformatics, Genomics and Computational Biology</li>
          
          <li>Biological Big Data Platform</li>
          
          <li>Artificial Intelligence Drug Discovery (AIDD) especially 
            Drug Target Identification and Drug Repurposing</li>
          
          <li>Modernization of Chinese herbal medicine</li>
          
        </ul>
      </div>
      

      
      <div class="col-sm-7">
        <h3>Education</h3>
        <ul class="ul-edu fa-ul">
          
          <li>
            <i class="fa-li fa fa-graduation-cap"></i>
            <div class="description">
              <p class="course">PhD in Biological Science, School of Medicine</p>
              <p class="institution">The Chinese University of Hong Kong, Shenzhen (CUHKSZ)</p>
            </div>
          </li>
          
          <li>
            <i class="fa-li fa fa-graduation-cap"></i>
            <div class="description">
              <p class="course">BSc in Ecology, School of Life Science, 2015</p>
              <p class="institution">Sun Yat-sen University (SYSU)</p>
            </div>
          </li>
          
        </ul>
      </div>
      

    </div>
  </div>
</div>

    </div>
  </section>

  
  
  <section id="publications_selected" class="home-section">
    <div class="container">
      



<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>Publications</h1>
    
  </div>
  <div class="col-xs-12 col-md-8">
    <p><ol>
<li><p>Junpeng Zhang <sup>+*</sup> , <strong>Taosheng Xu</strong> <sup>+</sup>, Lin Liu, Wu Zhang, Chunwen Zhao, Sijing Li, Jiuyong Li, Nini Rao<sup>*</sup>, Thuc Duy Le<sup>*</sup>. <em>LMSM: a modular approach for identifying lncRNA related miRNA sponge modules in breast cancer</em>. <strong>PLoS Computational Biology</strong> 16(4): e1007851  (2020). <br>
<strong>URL</strong>:<a href="https://doi.org/10.1371/journal.pcbi.1007851" target="_blank">https://doi.org/10.1371/journal.pcbi.1007851</a>  <br></p></li>

<li><p>Xiaomei Li, Lin Liu, Gregory J. Goodall, Andreas Schreiber, <strong>Taosheng Xu</strong>, Jiuyong Li, Thuc Duy Le<sup>*</sup>. <em>A novel single-cell based method for breast cancer prognosis</em>. <strong>PLoS Computational Biology</strong> 16(8): e1008133  (2020). <br>
<strong>URL</strong>:<a href="https://doi.org/10.1371/journal.pcbi.1008133" target="_blank">https://doi.org/10.1371/journal.pcbi.1008133</a> <br></p></li>

<li><p><strong>Taosheng Xu</strong>, Ning Su, Lin Liu, Junpeng Zhang, Hongqiang Wang, Weijia Zhang, Jie Gui, Kui Yu, Jiuyong Li, Thuc Duy Le<sup>*</sup>. <em>miRBaseConverter: An R/Bioconductor Package for Converting and Retrieving miRNA Name, Accession, Sequence and Family Information in Different Versions of miRBase</em>.  <strong>BMC Bioinformatics</strong> 19 (s19) :514 (2018).<br>
<strong>URL</strong>: <a href="https://doi.org/10.1186/s12859-018-2531-5" target="_blank">https://doi.org/10.1186/s12859-018-2531-5</a>  <br></p></li>

<li><p><strong>Taosheng Xu</strong><sup>+</sup>, Thuc Duy Le<sup>+*</sup>, Lin Liu, Ning Su, Rujing Wang, Bingyu Sun, Antonio Colaprico, Gianluca Bontempi, Jiuyong Li<sup>*</sup>. <em>CancerSubtypes: an R/Bioconductor package for molecular cancer subtype identification, validation, and visualization</em>. <strong>Bioinformatics</strong> 33(19): 3131–3133 (2017).<br>
<strong>URL</strong>: <a href="https://doi.org/10.1093/bioinformatics/btx378" target="_blank">https://doi.org/10.1093/bioinformatics/btx378</a> <br></p></li>

<li><p><strong>Taosheng Xu</strong><sup>+</sup>, Thuc Duy Le<sup>+*</sup>, Lin Liu, Rujing Wang, Bingyu Sun, Jiuyong Li<sup>*</sup>. <em>Identifying cancer subtypes from miRNA-TF-mRNA regulatory networks and expression data</em>. <strong>PLoS ONE</strong> 11(4) : e0152792 (2016).<br>
<strong>URL</strong>: <a href="https://doi.org/10.1371/journal.pone.0152792" target="_blank">https://doi.org/10.1371/journal.pone.0152792</a> <br></p></li>

<li><p>Sarvenaz Choobdar, et al., <strong>Taosheng Xu</strong> in DREAM Module Identification Challenge Consortium. <em>Assessment of network module identification across complex diseases</em>. <strong>Nature Methods</strong> 16: 843–852 (2019).<br>
<strong>URL</strong>:<a href="https://doi.org/10.1038/s41592-019-0509-5" target="_blank">https://doi.org/10.1038/s41592-019-0509-5</a> <br></p></li>

<li><p>Peter Brown, <strong>Taosheng Xu</strong> in RELISH Consortium , Yaoqi Zhou<sup>*</sup>. Large expert-curated database for benchmarking document similarity detection in biomedical literature search. <strong>Database</strong> 2019: baz085 (2019).
<strong>URL</strong>:<a href="https://doi.org/10.1093/database/baz085" target="_blank">https://doi.org/10.1093/database/baz085</a> <br></p></li>

<li><p>Junpeng Zhang<sup>*</sup>, Lin Liu, <strong>Taosheng Xu</strong>, Yong Xie, Chunwen Zhao, Jiuyong Li, Thuc Duy Le<sup>*</sup>. <em>miRspongeR: an R/Bioconductor package for the identification and analysis of miRNA sponge interaction networks and modules</em>. <strong>BMC Bioinformatics</strong> 20:235 (2019).<br>
<strong>URL</strong>: <a href="https://doi.org/10.1186/s12859-019-2861-y" target="_blank">https://doi.org/10.1186/s12859-019-2861-y</a> <br></p></li>

<li><p>Junpeng Zhang, Vu Viet Hoang Pham, Lin Liu, <strong>Taosheng Xu</strong>, Buu Truong, Jiuyong Li,Nini Rao<sup>*</sup>, Thuc Duy Le<sup>*</sup>. <em>Identifying miRNA synergism using multiple-intervention causal inference</em>. <strong>BMC Bioinformatics</strong> 20:613 (2019).<br>
<strong>URL</strong>:<a href="https://doi.org/10.1186/s12859-019-3215-5" target="_blank">https://doi.org/10.1186/s12859-019-3215-5</a> <br></p></li>

<li><p>Vu Viet Hoang Pham, Junpeng Zhang, Lin Liu, Buu Minh Thanh Truong, <strong>Taosheng Xu</strong>, Trung Tin Nguyen, Jiuyong Li,Thuc Duy Le<sup>*</sup>. <em>Identifying miRNA-mRNA regulatory relationships in breast cancer with invariant causal prediction</em>. <strong>BMC Bioinformatics</strong> 20:143 (2019).<br>
<strong>URL</strong>: <a href="https://doi.org/10.1186/s12859-019-2668-x" target="_blank">https://doi.org/10.1186/s12859-019-2668-x</a> <br></p></li>

<li><p>Thuc Duy Le<sup>*</sup>, <strong>Taosheng Xu</strong>, Lin Liu, Hu Shu, Tao Hoang, Jiuyong Li. <em>ParallelPC: An R Package for Efficient Causal Exploration in Genomic Data.</em>  In Pacific-Asia Conference on Knowledge Discovery and Data Mining (PAKDD 2018), pp.207-218. Springer.   (<a href="https://taoshengxu.github.io/DocumentGit/pdf/ParallelPC.pdf" target="_blank">Link</a>)<br>
<strong>URL</strong>: <a href="https://doi.org/10.1007/978-3-030-04503-6_22" target="_blank">https://doi.org/10.1007/978-3-030-04503-6_22</a> <br></p></li>

<li><p>Thuc Duy Le<sup>+*</sup>, Junpeng Zhang<sup>+</sup>, Lin Liu, Buu Minh Thanh Truong, Shu Hu, <strong>Taosheng Xu</strong>, Jiuyong Li<sup>*</sup>. <em>Identifying microRNA targets in epithelial-mesenchymal transition using joint-intervention causal inference.</em> In Proceedings of the 8th International Conference on Computational Systems-Biology and Bioinformatics (CSBio 2017), pp. 34-41. ACM. <br>
<strong>URL</strong>: <a href="https://doi.org/10.1145/3156346.3156353" target="_blank">https://doi.org/10.1145/3156346.3156353</a></p></li>

<li><p>Thuc Duy Le<sup>*</sup>, Lin Liu, <strong>Taosheng Xu</strong>, Junpeng Zhang, Jiuyong Li. <em>Abstract: A dry lab for exploring miRNA functions and applications in cancer subtype discovery</em>. The Third Asia-Pacific Bioconductor Meeting, Adelaide, Australia, 2017. (<a href="https://taoshengxu.github.io/DocumentGit/pdf/BioC2017.pdf" target="_blank">Link</a>)</p></li>

<li><p><strong>Taosheng Xu</strong><sup>*</sup>, Ning Su, Rujing Wang, Liangtu Song. <em>Gene selection for cancer clustering analysis based on expression data</em>. 2015 4th International Conference on Computer Science and Network Technology (ICCSNT 2015), pp. 516-519. IEEE.</p></li>

<li><p>Ning Su, <strong>Taosheng Xu</strong>, Liangtu Song<sup>*</sup>, Rujing Wang, Yuanyuan Wei. <em>Variable rate fertilization system with adjustable active feed-roll length</em>. International Journal of Agricultural and Biological Engineering(IJABE) 8, no. 4: 19-26 (2015). (IF: 1.007)<br>
<strong>URL</strong>: <a href="https://ijabe.org/index.php/ijabe/article/view/1644" target="_blank">https://ijabe.org/index.php/ijabe/article/view/1644</a> <br></p></li>
</ol>

<p><br></p>

<h2 id="software"><strong>Software</strong></h2>

<ol>
<li><p><strong>Taosheng Xu</strong>, Thuc Duy Le, <em>R/Bioconductor</em> package: <strong>miRBaseConverter</strong> (2017). <br>
<strong>URL</strong>: <a href="https://bioconductor.org/packages/miRBaseConverter/" target="_blank">https://bioconductor.org/packages/miRBaseConverter/</a> <br>
<strong>Online Tool</strong>:  <a href="http://nugget.unisa.edu.au:3838/miRBaseConverter" target="_blank">http://nugget.unisa.edu.au:3838/miRBaseConverter</a> or  <br> <a href="https://taoshengxu.shinyapps.io/miRBaseConverter/" target="_blank">https://taoshengxu.shinyapps.io/miRBaseConverter/</a></p></li>

<li><p><strong>Taosheng Xu</strong>, Thuc Duy Le, <em>R/Bioconductor</em> package: <strong>CancerSubtypes</strong> (2016). <br>
<strong>URL</strong>: <a href="https://bioconductor.org/packages/CancerSubtypes/" target="_blank">https://bioconductor.org/packages/CancerSubtypes/</a></p></li>
</ol>
</p>

    
    <ul class="fa-ul">
      
    </ul>
    

  </div>
</div>

    </div>
  </section>

  
  
  <section id="publications_recent" class="home-section">
    <div class="container">
      




<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>Publication Details</h1>
    
    
  </div>
  <div class="col-xs-12 col-md-8">
    

    
    
    <div class="pub-list-item" itemscope itemtype="http://schema.org/CreativeWork">
  <div class="row">

    

    <div class="col-md-12">

    

      <h3 class="article-title" itemprop="name">
        <a href="https://taoshengxu.github.io/publication/mirbaseconverter_paper/" itemprop="url">miRBaseConverter: an R/Bioconductor package for converting and retrieving miRNA name, accession, sequence and family information in different versions of miRBase</a>
      </h3>

      <div class="pub-abstract" itemprop="text">
        
        We introduce miRBaseConverter, an R package integrating the latest miRBase version 22 available in Bioconductor to provide a suite of functions for converting and retrieving miRNA name (ID), accession, sequence, species, version and family information in different versions of miRBase. The package is implemented in R and available under the GPL-2 license from the Bioconductor website (http://bioconductor.org/packages/miRBaseConverter/). A Shiny-based GUI suitable for non-R users is also available as a standalone application from the package and also as a web application at http://nugget.unisa.edu.au:3838/miRBaseConverter. miRBaseConverter has a built-in database for querying miRNA information in all species and for both pre-mature and mature miRNAs defined by miRBase. In addition, it is the first tool for batch querying the miRNA family information. The package aims to provide a comprehensive and easy-to-use tool for miRNA research community where researchers often utilize published miRNA data from different sources.
        
      </div>

      <div class="pub-authors" itemprop="author">
        
        Taosheng Xu, Ning Su, Lin Liu, Junpeng Zhang, Hongqiang Wang, Weijia Zhang, Jie Gui, Kui Yu, Jiuyong Li, Thuc Duy Le
        
      </div>

      <div class="pub-publication">
        
        In <em>BMC Bioinformatics</em>
        
      </div>

      <div class="pub-links">
        



<a class="btn btn-primary btn-outline btn-xs" href="https://taoshengxu.github.io/publication/mirbaseconverter_paper/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://bmcbioinformatics.biomedcentral.com/track/pdf/10.1186/s12859-018-2531-5">
  PDF
</a>







<a class="btn btn-primary btn-outline btn-xs" href="#ZgotmplZ">
  Online
</a>


      </div>

    </div>
  </div>
</div>

    
    <div class="pub-list-item" itemscope itemtype="http://schema.org/CreativeWork">
  <div class="row">

    

    <div class="col-md-12">

    

      <h3 class="article-title" itemprop="name">
        <a href="https://taoshengxu.github.io/publication/mirbaseconverter_package/" itemprop="url">[R package]miRBaseConverter: A comprehensive and high-efficiency tool for converting and retrieving the information of miRNAs in different miRBase versions</a>
      </h3>

      <div class="pub-abstract" itemprop="text">
        
        
        
      </div>

      <div class="pub-authors" itemprop="author">
        
        Taosheng Xu, Thuc Duy Le
        
      </div>

      <div class="pub-publication">
        
        In <em>Bioconductor</em>
        
      </div>

      <div class="pub-links">
        



<a class="btn btn-primary btn-outline btn-xs" href="https://taoshengxu.github.io/publication/mirbaseconverter_package/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://bioconductor.org/packages/devel/bioc/manuals/miRBaseConverter/man/miRBaseConverter.pdf">
  PDF
</a>




<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/xtsvm/miRBaseConverter">
  Code
</a>




<a class="btn btn-primary btn-outline btn-xs" href="https://bioconductor.org/packages/devel/bioc/vignettes/miRBaseConverter/inst/doc/miRBaseConverter-vignette.html">
  Vignette
</a>

<a class="btn btn-primary btn-outline btn-xs" href="https://bioconductor.org/packages/devel/bioc/html/miRBaseConverter.html">
  Online
</a>


      </div>

    </div>
  </div>
</div>

    
    <div class="pub-list-item" itemscope itemtype="http://schema.org/CreativeWork">
  <div class="row">

    

    <div class="col-md-12">
      <a href="https://taoshengxu.github.io/publication/cancersubtypes_paper/">
        <img src="/img/CancerSubtypes_Framework.jpg" class="pub-banner"
             itemprop="image">
      </a>
    </div>
    <div class="col-md-12">

    

      <h3 class="article-title" itemprop="name">
        <a href="https://taoshengxu.github.io/publication/cancersubtypes_paper/" itemprop="url">CancerSubtypes: an R/Bioconductor package for molecular cancer subtype identification, validation and visualization</a>
      </h3>

      <div class="pub-abstract" itemprop="text">
        
        Identifying molecular cancer subtypes from multi-omics data is an important step in the personalized medicine. We introduce CancerSubtypes, an R package for identifying cancer subtypes using multi-omics data, including gene expression, miRNA expression and DNA methylation data. CancerSubtypes integrates four main computational methods which are highly cited for cancer subtype identification and provides a standardized framework for data pre-processing, feature selection, and result follow-up analyses, including results computing, biology validation and visualization. The input and output of each step in the framework are packaged in the same data format, making it convenience to compare different methods. The package is useful for inferring cancer subtypes from an input genomic dataset, comparing the predictions from different well-known methods and testing new subtype discovery methods, as shown with different application scenarios in the Supplementary Material.
        
      </div>

      <div class="pub-authors" itemprop="author">
        
        Taosheng Xu, Thuc Duy Le, Lin Liu, Ning Su, Rujing Wang, Bingyu Sun, Antonio Colaprico, Gianluca Bontempi, Jiuyong Li
        
      </div>

      <div class="pub-publication">
        
        In <em>Bioinformatics</em>
        
      </div>

      <div class="pub-links">
        



<a class="btn btn-primary btn-outline btn-xs" href="https://taoshengxu.github.io/publication/cancersubtypes_paper/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://taoshengxu.github.io/pdf/Xu_CancerSubtypes.pdf">
  PDF
</a>







<a class="btn btn-primary btn-outline btn-xs" href="https://taoshengxu.github.io/pdf/btx378_Supp.pdf">
  Supplementary Material
</a>

<a class="btn btn-primary btn-outline btn-xs" href="https://doi.org/10.1093/bioinformatics/btx378">
  Online
</a>


      </div>

    </div>
  </div>
</div>

    
    <div class="pub-list-item" itemscope itemtype="http://schema.org/CreativeWork">
  <div class="row">

    

    <div class="col-md-12">

    

      <h3 class="article-title" itemprop="name">
        <a href="https://taoshengxu.github.io/publication/cancersubtypes_package/" itemprop="url">[R package]CancerSubtypes: Cancer subtypes identification, validation and visualization based on genomic data</a>
      </h3>

      <div class="pub-abstract" itemprop="text">
        
        
        
      </div>

      <div class="pub-authors" itemprop="author">
        
        Taosheng Xu, Thuc Duy Le
        
      </div>

      <div class="pub-publication">
        
        In <em>Bioconductor</em>
        
      </div>

      <div class="pub-links">
        



<a class="btn btn-primary btn-outline btn-xs" href="https://taoshengxu.github.io/publication/cancersubtypes_package/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://bioconductor.org/packages/release/bioc/manuals/CancerSubtypes/man/CancerSubtypes.pdf">
  PDF
</a>




<a class="btn btn-primary btn-outline btn-xs" href="https://github.com/xtsvm/CancerSubtypes">
  Code
</a>




<a class="btn btn-primary btn-outline btn-xs" href="https://bioconductor.org/packages/release/bioc/vignettes/CancerSubtypes/inst/doc/CancerSubtypes-vignette.html">
  Vignette
</a>

<a class="btn btn-primary btn-outline btn-xs" href="https://bioconductor.org/packages/release/bioc/html/CancerSubtypes.html">
  Online
</a>


      </div>

    </div>
  </div>
</div>

    
    <div class="pub-list-item" itemscope itemtype="http://schema.org/CreativeWork">
  <div class="row">

    

    <div class="col-md-12">
      <a href="https://taoshengxu.github.io/publication/mirtfmrna_paper/">
        <img src="/img/WSNF.jpg" class="pub-banner"
             itemprop="image">
      </a>
    </div>
    <div class="col-md-12">

    

      <h3 class="article-title" itemprop="name">
        <a href="https://taoshengxu.github.io/publication/mirtfmrna_paper/" itemprop="url">Identifying Cancer Subtypes from miRNA-TF-mRNA Regulatory Networks and Expression Data</a>
      </h3>

      <div class="pub-abstract" itemprop="text">
        
        In this paper, we propose a method, weighted similarity network fusion (WSNF), to utilise the information in the complex miRNA-TF-mRNA regulatory network in identifying cancer subtypes. We firstly build the regulatory network where the nodes represent the features, i.e. the microRNAs (miRNAs), transcription factors (TFs) and messenger RNAs (mRNAs) and the edges indicate the interactions between the features. The interactions are retrieved from various interatomic databases. We then use the network information and the expression data of the miRNAs, TFs and mRNAs to calculate the weight of the features, representing the level of importance of the features. The feature weight is then integrated into a network fusion approach to cluster the samples (patients) and thus to identify cancer subtypes. We applied our method to the TCGA breast invasive carcinoma (BRCA) and glioblastoma multiforme (GBM) datasets. The experimental results show that WSNF performs better than the other commonly used computational methods, and the information from miRNA-TF-mRNA regulatory network contributes to the performance improvement. The WSNF method successfully identified five breast cancer subtypes and three GBM subtypes which show significantly different survival patterns. We observed that the expression patterns of the features in some miRNA-TF-mRNA sub-networks vary across different identified subtypes. In addition, pathway enrichment analyses show that the top pathways involving the most differentially expressed genes in each of the identified subtypes are different. The results would provide valuable information for understanding the mechanisms characterising different cancer subtypes and assist the design of treatment therapies. All datasets and the R scripts to reproduce the results are available online at the website: http://nugget.unisa.edu.au/Thuc/cancersubtypes/.
        
      </div>

      <div class="pub-authors" itemprop="author">
        
        Taosheng Xu, Thuc Duy Le, Lin Liu, Rujing Wang, Bingyu Sun, Jiuyong Li
        
      </div>

      <div class="pub-publication">
        
        In <em>PLoS ONE</em>
        
      </div>

      <div class="pub-links">
        



<a class="btn btn-primary btn-outline btn-xs" href="https://taoshengxu.github.io/publication/mirtfmrna_paper/">
  Details
</a>


<a class="btn btn-primary btn-outline btn-xs" href="https://taoshengxu.github.io/pdf/Xu_2016_Identifying%20Cancer%20Subtypes%20from%20miRNA-TF-mRNA%20Regulatory%20Netw.pdf">
  PDF
</a>





<a class="btn btn-primary btn-outline btn-xs" href="http://nugget.unisa.edu.au/Thuc/cancersubtypes/">
  Dataset
</a>



<a class="btn btn-primary btn-outline btn-xs" href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0152792">
  Online
</a>


      </div>

    </div>
  </div>
</div>

    
    

  </div>
</div>

    </div>
  </section>

  
  
  <section id="posts" class="home-section">
    <div class="container">
      




<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">

    <h1>Awards</h1>
    
    

  </div>
  <div class="col-xs-12 col-md-8">

    <p><h2 id="dream-11-2016-all-stars-http-dreamchallenges-org-dream-11-2016-best-performers"><a href="http://dreamchallenges.org/dream-11-2016-best-performers/" target="_blank">DREAM 11- 2016 All-Stars</a></h2>

<p>Causality Team: Weijia Zhang, Thuc Le, <strong>Taosheng Xu</strong>, Junpeng Zhang, Lin Liu, Jiuyong Li<br />
<em>Honorable Mention</em> of Disease Module Identification DREAM Challenge (Sub-Challenge 1)</p>

    
      
    

  </div>
</div>

    </div>
  </section>

  
  
  <section id="contact" class="home-section">
    <div class="container">
      




<div class="row">
  <div class="col-xs-12 col-md-4 section-heading">
    <h1>Contact</h1>
    
  </div>
  <div class="col-xs-12 col-md-8">
    

    <ul class="fa-ul" itemscope>

      
      <li>
        <i class="fa-li fa fa-envelope fa-2x" aria-hidden="true"></i>
        <span id="person-email" itemprop="email"><a href="mailto:taosheng.x%20at%20gmail.com">taosheng.x at gmail.com</a></span>
      </li>
      

      

      

      

      

      
      <li>
        <i class="fa-li fa fa-map-marker fa-2x" aria-hidden="true"></i>
        <span id="person-address" itemprop="address">Shenzhen, China</span>
      </li>
      

      

    </ul>

  </div>
</div>

    </div>
  </section>



<footer class="site-footer">


  <div class="container">
    <p class="powered-by">

      &copy; 2017-2020 Taosheng Xu &middot; 

      Powered by the <a href="https://github.com/gcushen/hugo-academic" target="_blank">Academic theme</a> for <a href="http://gohugo.io" target="_blank">Hugo</a>.

      <span class="pull-right" aria-hidden="true">
        <a href="#" id="back_to_top">
          <span class="button_icon">
            <i class="fa fa-chevron-up fa-2x"></i>
          </span>
        </a>
      </span>

    </p>
  </div>
</footer>






    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.12.4/jquery.min.js" integrity="sha512-jGsMH83oKe9asCpkOVkBnUrDDTp8wl+adkB2D+//JtlxO4SrLoJdhbOysIFQJloQFD+C4Fl1rMsQZF76JjV0eQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.2/imagesloaded.pkgd.min.js" integrity="sha512-iHzEu7GbSc705hE2skyH6/AlTpOfBmkx7nUqTLGzPYR+C1tRaItbRlJ7hT/D3YQ9SV0fqLKzp4XY9wKulTBGTw==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/1.19.1/TweenMax.min.js" integrity="sha512-Z5heTz36xTemt1TbtbfXtTq5lMfYnOkXM2/eWcTTiLU01+Sw4ku1i7vScDc8fWhrP2abz9GQzgKH5NGBLoYlAw==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/gsap/1.19.1/plugins/ScrollToPlugin.min.js" integrity="sha512-CDeU7pRtkPX6XJtF/gcFWlEwyaX7mcAp5sO3VIu/ylsdR74wEw4wmBpD5yYTrmMAiAboi9thyBUr1vXRPA7t0Q==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha512-iztkobsvnjKfAtTNdHkGVjAYTrrtlC7mGp/54c40wowO7LhURYl3gVzzcEqGl/qKXQltJ2HwMrdLcNUdo+N/RQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.4/isotope.pkgd.min.js" integrity="sha512-VDBOIlDbuC4VWxGJNmuFRQ0Li0SKkDpmGyuhAG5LTDLd/dJ/S0WMVxriR2Y+CyPL5gzjpN4f/6iqWVBJlht0tQ==" crossorigin="anonymous"></script>
    
    <script src="/js/hugo-academic.js"></script>
    

    
    
      
      <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.9.0/highlight.min.js"></script>

      

      

      <script>hljs.initHighlightingOnLoad();</script>
    

    
    

  </body>
</html>

